README.md

```diff
@@ -25,6 +25,12 @@ To run a local zip file through the checks:
 
 `gscan /path/to/theme.zip -z`
 
+This can also be used to integrate gscan into a continuous integration.
+Gscan exits with
+  - 0 if there are no errors or warnings
+  - 1 on errors
+  - 2 on warnings
```

Comment:
Not sure about the status code "2" here 🤔 

Please take a look at e.g. [this link](http://tldp.org/LDP/abs/html/exitcodes.html), which describes the meanings of exit codes.